.travis.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - env:
         - USER_EMAIL=olafurpg@gmail.com
       stage: release
-      script: sbt docusaurusPublishGhpages
+      script: sbt docs/docusaurusPublishGhpages
```

Comment:
Have you checked if this works? The 0.1.2 release didn't do changes here.

Reply:
nope, I'm using this as a placeholder. I suspect 0.1.2 isn't published yet, is it?

Reply:
It's already released, might take a while to sync to Maven Central